parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -87,11 +87,11 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("statemine"),
 	impl_name: create_runtime_str!("statemine"),
 	authoring_version: 1,
-	spec_version: 9300,
+	spec_version: 9301,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 8,
-	state_version: 0,
+	state_version: 1,
```

Comment:
We should feature gate this for now. 

As the GCP is still deploying the chain on Rococo for testing.